blueprints/automation/unifiprotect/push_notification_smart_event.yaml

```diff
@@ -16,6 +16,7 @@ blueprint:
 
       - Precense filter for only sending notifications for when you are not home
       - [HTML5 Push Notification Target][1] and/or [Mobile App Notification Target][2] as well as toggles for both
+      - Telegram Notification option, including toggle
```

Comment:
The description is markdown whereas the help text options below are _not_. So, you should add a link to the Telegram documentation here, similar to all of the other ones.